api_testing/apis/test_unit_api.py

```diff
@@ -320,4 +320,14 @@ def get_testunit_with_empty_specification(self):
         for testunit in testunits:
             if testunit['specifications'] == '':
                 testunits_name.append(testunit['name'])
-        return testunits_name
\ No newline at end of file
+        return testunits_name
+
+    def get_test_unit_with_spec_or_quan_only(self, spec_or_quan):
+        testunits = self.get_all_test_units(filter='{"typeName":2}').json()['testUnits']
+        for testunit in testunits:
+            if spec_or_quan == 'spec' and testunit['specifications'] != '' and testunit['quantification'] == '':
+                return testunit['id']
+                break
+            elif spec_or_quan == 'quan' and testunit['specifications'] == '' and testunit['quantification'] != '':
+                return testunit['id']
+                break
```

Comment:
there is no need to `break` after `return`

Reply:
I updated it , thank you